File: code/frontend/src/app/models/dto/FinanceiroMensalDTO.ts
import { AcumuladoFinanceiroDTO } from "./AcumuladoFinanceiroDTO";

export interface FinanceiroMensalDTO {
  mes: string;
  totalVendas: number;
  totalValorVendas: number;
  totalLucro: number;
  acumuladoAno: AcumuladoFinanceiroDTO;
}

File: code/backend/src/main/java/br/com/anagropets/dto/financeiro/FinanceiroMensalDTO.java
package br.com.anagropets.dto.financeiro;

import java.math.BigDecimal;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class FinanceiroMensalDTO {

    private String mes;
    private Long totalVendas;
    private BigDecimal totalValorVendas;
    private BigDecimal totalLucro;
    private AcumuladoFinanceiroDTO acumuladoAno;
}
